cumulusci/tasks/salesforce/create_permission_sets.py

```diff
@@ -0,0 +1,35 @@
+from cumulusci.tasks.salesforce.BaseSalesforceApiTask import BaseSalesforceApiTask
+from cumulusci.core.utils import process_list_arg
+
+
+class CreatePermissionSet(BaseSalesforceApiTask):
+    task_options = {
+        "api_name": {
+            "description": "API names of generated Permission Set",
+            "required": True,
+        },
+        "label": {"description": "Label of generated Permission Set"},
+        "user_permissions": {
+            "description": "List of User Permissions to include in the Permission Set.",
+            "required": True,
+        },
+    }
+
+    def _init_options(self, kwargs):
+        super()._init_options(kwargs)
+
+        self.options["user_permissions"] = process_list_arg(
+            self.options["user_permissions"]
+        )
+
+    def _run_task(self):
+        result = self.sf.PermissionSet.create(
+            {
+                "Name": self.options["api_name"],
+                "Label": self.options.get("label") or self.options["api_name"],
+                **{perm: True for perm in self.options["user_permissions"]},
+            }
+        )
+        result = self.sf.PermissionSetAssignment.create(
```

Comment:
If we aren't referencing `result` do we need to assign it a new value here? Also, if there's an error creating either the PS or PSA I'm guessing that SFDX is bubbling up the message to the user?

Reply:
We don't, I'll fix that.

Originally I had written error-handling code here but I quickly found that any mishap in creation just results in simple_salesforce throwing an exception rather than me getting back a message.